### DISCUSSION TITLE
vuepress-plugin-search-pro 搜索不到标题中的 <code>

### DISCUSSION THREAD

**Discussion by Dilant** [vuepress-theme-hope/vuepress-theme-hope#2875] [Q&A] [answered]
我正在 Vuepress Next 的默认主题上使用 `vuepress-plugin-search-pro` 本地搜索插件。

我注意到，标题中内嵌的 `<code>` 内容无法被搜索到，像这样：

### 标题中的代码 `def some_function(x: str) -> bool`

```md
### 标题中的代码 `def some_function(x: str) -> bool`
```

此时搜索 `some_function` 没有搜索结果。我这里有大量类似的写法（[页面范例](https://github.com/Dilant/B2METutorial/blob/master/src/advanced/executor.md)），因此希望能够支持搜索到标题中内嵌 `<code>` 的内容。

<details>

<summary>系统环境</summary>

- 系统：Windows 11
- Node：`19.7.0`
- pnpm：`7.27.1`
- 依赖：

```json
"devDependencies": {
  "@vuepress/client": "2.0.0-beta.61",
  "@vuepress/plugin-shiki": "2.0.0-beta.61",
  "check-md": "github:Dilant/check-md#master",
  "prettier": "^2.8.4",
  "vue": "^3.2.47",
  "vuepress": "2.0.0-beta.61",
  "vuepress-plugin-search-pro": "2.0.0-beta.189"
}
```

</details>

**Comment by Mister-Hope** [comment, marked as answer]
开个issue吧，回头看看
